controller/control.go

```diff
@@ -295,14 +333,8 @@ func (c *Controller) addQuorumReplica(address string, snapshot bool) error {
 		return fmt.Errorf("Failed to set rebuild : %v", true)
 	}
 	c.UpdateVolStatus()
+	c.UpdateCheckpoint()
```

Comment:
I think we should remove Quorum related code.

Reply:
agreed, this is a long pending task which need to be completed.